Source/gg2/Objects/InGameElements/Character.events/User Event 7.xml

```diff
@@ -17,10 +17,13 @@
 // For Infiltrators only.
 var ghost;
 ghost = instance_create(x, y, SpyGhost);
-ghost.image_index = floor(animationImage);
+ghost.image_index = 0;//floor(animationImage);
```

Comment:
Nope. The sprites don't match up anymore, it looks like, so I just made it display the default sprite image in the ghost sprite instead.